compiler/compiler.ts

```diff
@@ -300,6 +300,9 @@ export class Compiler {
     static MAX_INT_32 = 2_147_483_647;
     static MIN_INT_32 = -2_147_483_648;
 
+    static MIN_INT_64 = -9223372036854775808n;
```

Comment:
In this case, I prefer having an actual number I can look at , instead of having to evaluate an expression first.
Might not be too relevant for big numbers like MIN_INT_64, but then it is consistent at least.